[pull] primary from flybywiresim:primary (#319)
Valastiri/fbw-userguide: docs/fbw-a32nx/feature-guides/cFMS.md

## Special Notes

### Flight Plan Sync (MSFS ATC + VFR Map)

> Our custom FMS provides better accuracy and features over the default offering in MSFS which results in issues syncing the flight plan from the MCDU back into the simulator. Flight plans with complex routing may have significant issues if synced backwards to MSFS's simplified flight planning. This will always be difficult unless Asobo improves the built-in flight planner. Other aircraft with complex flight planning capabilities can share this limitation.

Reviewer: ```suggestion
Our custom FMS provides better accuracy and features over the default offering in MSFS which results in issues syncing the flight plan from the MCDU back into the simulator. Flight plans with complex routing may have significant issues if synced backwards to MSFS's simplified flight planning. This will always be problematic unless Asobo improves the built-in flight planner. Other aircraft with complex flight planning capabilities also have this limitation.
```

!!! warning "VFR Map"
    We'd like to stress that the VFR Map is not supported and we don't recommend using it - the A320neo is an IFR aircraft after all.

>     When flight plan sync is enabled you will not see the proper sequence of waypoints or pathing between waypoints. 

Reviewer: ```suggestion
    When flight plan sync is enabled you will often not see the proper sequence of waypoints or pathing between waypoints. 
```

    **The VFR Map will attempt to you show you what the simulator's built-in flight planner produces**, which is not an accurate representation of the LNAV in our cFMS. In the future we may replicate a secondary route visualization (apart from the PLAN mode on the ND) feature in the EFB.

> As noted above ^^*your mileage may vary*^^ but we have provided a feature on the EFB that helps sync the flight plan so MSFS ATC can read it. By default, this feature is set to `None` on the EFB.

Reviewer: ```suggestion
With the above in mind we have provided a feature that helps sync the aircraft's flight plan back to the MSFS' built-in flight plan. This feature can be configured on the EFB and is set to `None` by default.
```

Reviewer 2: modified this sentence a tiny bit
